tests-e2e/argocd/argocd_application_test.go

```diff
@@ -23,7 +25,7 @@ var _ = Describe("Argo CD Application", func() {
 			Expect(fixture.EnsureCleanSlate()).To(Succeed())
 		})
 
-		It("Argo CD Application should have has prune, allowEmpty and selfHeal enabled", func() {
+		It("Argo CD Application should have has prune, allowEmpty and selfHeal enabled and verify whether AppProject resource has been created", func() {
```

Comment:
The purpose of this test is just to check prune, allowEmpty, and selfHeal. I think verifying the AppProject would be better handled in other E2E tests. I've added some comments related to this elsewhere.